domain/modeldefaults/state/state.go

```diff
@@ -55,29 +88,35 @@ func (s *State) CloudDefaults(
 
 	db, err := s.DB()
 	if err != nil {
-		return rval, errors.Trace(err)
+		return rval, errors.Capture(err)
```

Comment:
Oh, so we're calling it Capture.
What was wrong with Trace which is a well understood paradigm for the Juju code base? Seems like a lot of churn; will it pay its way?